books/architecture/src/rpc/differences/json-strictness.md

```diff
@@ -0,0 +1,43 @@
+# JSON strictness
+This is a list of behavior that `monerod`'s JSON parser allows, that Cuprate's JSON parser ([`serde_json`](https://docs.rs/serde_json)) does not.
+
+In general, `monerod`'s parser is quite lenient, allowing invalid JSON in many cases.
+Cuprate's (really, `serde_json`) JSON parser is quite strict, essentially sticking to
+the [JSON specification](https://datatracker.ietf.org/doc/html/rfc8259).
+
+Cuprate also makes some decisions that are _different_ than `monerod`, but are not necessarily more or less strict.
+
+## Missing closing bracket
+`monerod` will accept JSON missing a final closing `}`.
+
+Example:
+```bash
+curl \
+	http://127.0.0.1:18081/json_rpc \
+	-d '{"jsonrpc":"2.0","id":"0","method":"get_block_count"' \
+	-H 'Content-Type: application/json'
+```
+
+## Trailing ending comma
+`monerod` will accept JSON containing a final trailing `,`.
+
+Example:
+```bash
+curl \
+	http://127.0.0.1:18081/json_rpc \
+	-d '{"jsonrpc":"2.0","id":"0","method":"get_block_count",}' \
+	-H 'Content-Type: application/json'
+```
+
+## Extra fields
+`monerod` will ignore extra fields within JSON from the user.
+
+Cuprate will return an error if any unknown field is present.
```

Comment:
I'm not sure if this is a good idea, for extensibility unknown fields are often ignored and serde will do this by default.

Why should we change this?

Reply:
Hmm sorry removed, I thought this was the default for us for some reason.